components/file_system/src/lib.rs

```diff
@@ -17,66 +18,17 @@ pub use std::fs::{
     FileType, Metadata, Permissions, ReadDir,
 };
 
-use std::cell::Cell;
 use std::io::{self, ErrorKind, Read, Write};
 use std::path::Path;
 use std::sync::{Arc, Mutex};
 
+pub use iosnoop::{
+    flush_io_metrics, get_io_type, init_io_snooper, set_io_type, IOContext, IOOp, IOType,
+    WithIOType,
+};
```

Comment:
I'm not familiar with organizing rust module, but I don't like defining these in a subcrate, could we at least get `IOOp, IOType, WithIOType` out?